stubs/inertia/resources/js/Jetstream/Select.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <select class="form-select w-full rounded-md shadow-sm" :value='value' @input="$emit('input', $event.target.value)" ref="select">
+        <option v-if='placeholder' :value='null' disabled>{{ placeholder }}</option>
```

Comment:
```suggestion
        <option v-if="placeholder" :value="null" disabled>{{ placeholder }}</option>
```

Reply:
I think `placeholder` prop is not needed - we can use `<slot></slot>` for that.

Reply:
IMO it's nice to have built in, and if you want to use the `<slot>` for that you still can, it doesn't prevent you in any way.

Reply:
Gah sorry, I thought StyleCI would catch all the quotes, etc.